runbooks/patterns/for-loops/for-retry-in-iteration.runbook.md

```
@@ -0,0 +1,25 @@
+  - for-loops
+scenarios:
+  after-retry:
+    description: Each iteration fails once then passes on retry
+    commands:
+      - rd run for-retry-in-iteration.runbook.md
+    result: COMPLETE
+---
+
+# FOR Retry In Iteration
+
+## 1. Process items
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)